lib/internal/bootstrap/pre_execution.js

```diff
@@ -132,6 +132,9 @@ function initializeReport() {
 
 function setupDebugEnv() {
   require('internal/util/debuglog').initializeDebugEnv(process.env.NODE_DEBUG);
+  if (getOptionValue('--expose-internals')) {
```

Comment:
We could actually consider throwing a warning here..and expose `internalBinding` automatically to the global proxy?
cc @Fishrock123

Reply:
I think leaving it in internal/test is fine